node_modules/gh-users/tests/test-search.js

```diff
@@ -226,4 +226,67 @@ describe('Users - Search', function() {
             });
         });
     });
+
+    /**
+     * Assert whether personal information should be returned from the get/search users feed
+     *
+     * @param  {Object}         actor                   The object that holds the user and client to fetch the users
+     * @param  {Number}         app                     The id of the application to search in
+     * @param  {String}         [query]                 The query to filter users by
+     * @param  {Boolean}        expectPersonalInfo      Whether user objects should have personal information
+     * @param  {Function}       callback                Standard callback function
+     */
+    var expectUsers = function(actor, app, query, expectPersonalInfo, callback) {
```

Comment:
I don't think so, its use is limited to this test file and it would make the `assertGetUsers` method signature a bit bloated I think